mlrun/datastore/azure_blob.py

```diff
@@ -71,6 +93,82 @@ def get_storage_options(self):
         )
         return self._sanitize_storage_options(res)
 
+    def do_connect(self, storage_options):
+        """Connect to the BlobServiceClient, using user-specified connection details.
```

Comment:
Seems like implementation details that shouldn't be in the docstring.

Reply:
fixed to comments.

Reply:
Seems like this one is still relevant. `BlobServiceClient` is an implementation detail. It's also not technically correct to say that we connect to the client.

Reply:
To me - fix to connect to azure without implementation details.